README.md

```diff
@@ -67,14 +68,15 @@ There is a top level chart-of-charts that will just install everything as a big
 ```shell
 helm repo add unikorn-cloud-capi https://unikorn-cloud.github.io/helm-cluster-api
 helm repo update
-helm install unikorn-cloud-capi/cluster-api --version v0.1.1
+helm install cluster-api unikorn-cloud-capi/cluster-api --version v0.2.0
```

Comment:
Note to self: history repeating itself... we need a way to not have to change versions all over the place and every time something changes.  Now asciidoc is good in that it has attributes, so that's a single place, markdown less so.  Ideally we'd like the source of truth to be the chart version.  So I guess the proposal would be to have a pre-commit hook that looks at `charts/*/Chart.yaml` and generates a `README.md` from a template, modifying in place just isn't going to fly given most of these things live in verbatim sections so markup of these things is impossible.

Reply:
Inspiration?
https://github.com/kubernetes-sigs/image-builder/blob/main/Makefile#L32-L34

Reply:
sed... sounds a bit flaky 🤣

Reply:
aaah yeah, just looked at that script (for probably the first time!).

ok yes, probably some parsing of the `Chart.yeaml` in the latest release for this would be sufficient then.